.github/workflows/code-quality.yml

```diff
@@ -36,15 +36,12 @@ jobs:
       - name: Update rust
         run: rustup update
 
-      - name: Install cargo-machete
-        run: cargo install cargo-machete
```

Comment:
Why is it redundant?

Reply:
I might be wrong. As you commented offline, it may have been already installed and be in the cache. Let's err on the safe side and continue on error, as you suggested

Reply:
Done at 1e6863d4